src/components/modules/MediumVideo.svelte

```diff
@@ -31,14 +31,14 @@
 </script>
 
 <div class="medium" id={medium.id}>
-  <SvgOverlay classes={classes} selected={selected} annotations={annotations} src={src.replace(/\.[^.]+$/, ".txt")}>
+  <!-- <SvgOverlay classes={classes} selected={selected} annotations={annotations} src={src.replace(/\.[^.]+$/, ".txt")}> -->
   {#if src.includes("mp4") || src.includes("mov")}
       <video controls muted {src} type="video/mp4" style:height="100%"/>
   {:else if src.includes("png") || src.includes("jpeg") || src.includes("jpg")}
       <!-- svelte-ignore a11y-missing-attribute -->
       <img {src} style:height="100%"/>
     {/if}
-  </SvgOverlay>
+  <!-- </SvgOverlay> -->
```

Comment:
The SVG needs a complete CSS review, as the dimensions of the iframe seem to break the current style setup. I was hoping to take that in the next PR, so we can keep making backend changes with the current elements.

Reply:
to do this you need to remove the SVG commits, at least.
and we need this done before we can consider this feature ready.